packages/design-system-react/src/components/temp-components/Maskicon/Maskicon.tsx

```diff
@@ -0,0 +1,41 @@
+import DOMPurify from 'dompurify';
+import React, { useEffect, useState } from 'react';
+
+import { MaskiconProps } from './Maskicon.types';
+import { getMaskiconSVG } from './Maskicon.utilities';
+
+export const Maskicon = ({
+  address,
+  size = 32,
+  style,
+  ...props
+}: MaskiconProps) => {
+  const [svgString, setSvgString] = useState('');
+
+  useEffect(() => {
+    let cancelled = false;
+    (async () => {
+      const newSvg = await getMaskiconSVG(address, size);
+      if (!cancelled) {
+        setSvgString(newSvg);
+      }
+    })();
+    return () => {
+      cancelled = true;
+    };
+  }, [address, size]);
+
+  if (!svgString) {
+    return <div style={{ width: size, height: size, ...style }} />;
+  }
```

Comment:
Do we want to provide any further error handling here? e.g. `onError` callback? Or do we want to add a default color via a classname?

Reply:
Ok sounds good